src/templates/actor/tabs/InventoryTab.html

```diff
@@ -43,7 +43,7 @@
                 </div>
             </div>
         </div>
-        <div class="scroll-area-inventory">
+        <div class="scroll-area scroll-area-inventory">
```

Comment:
Did you continue work on your previously merged branch?

If so: Please, in the future, refresh your master and create a new branch for each new PR, to avoid changes like these.